Modules/Devices/src/Devices.Domain/Entities/Identities/Identity.cs

```diff
@@ -76,6 +76,24 @@ public IdentityDeletionProcess StartDeletionProcessAsOwner(DeviceId asDevice)
         return deletionProcess;
     }
 
+    public void DeletionGracePeriodReminder1Sent()
+    {
+        var deletionProcess = DeletionProcesses.First(d => d.IsActive());
```

Comment:
What if we don't have an active deletion process?
This is a public method, so you can't assume it will only be called by your handler where you know there's an active process.